src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos, todosFilter } from './api/todos';
+import { FilterBy } from './types/FilterBy';
+import { NewTodoInput } from './components/NewTodoInput';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodosFilter } from './components/TodosFilter';
+import { ErrorMessage } from './types/ErrorMessage';
+import { NotificationError } from './components/NotificationError';
 
-const USER_ID = 0;
+const USER_ID = 6986;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(FilterBy.All);
+  const [error, setError] = useState<ErrorMessage | null>(null);
+
+  const activeTodosCount = todosFilter(todos, FilterBy.ACTIVE).length;
+  const completedTodosCount = todosFilter(todos, FilterBy.COMPLETED).length;
+  const isThereAreActiveTodos = todos.length === completedTodosCount;
+
+  const getTodosFromServer = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+      setVisibleTodos([...todosFromServer]);
+    } catch {
+      setError(ErrorMessage.DOWNLOAD);
+    }
```

Comment:
consider handling loading procces and show it on the UI (some spinner or smth like that)